### DISCUSSION TITLE
[Question]: Driveway camera placement

### DISCUSSION THREAD

**Discussion by jamesonuk** [blakeblackshear/frigate#12308] [Ask A Question]
### What is your question:

I currently have a Ring Wired Cam Plus covering my drive and want to replace this with something using Frigate for person and car detection.  Currently the camera is roughly in the middle of the house pointing forwards at about 3m.  I am drowning in the number of options and conflicting info out there and looking for a bit of advice.  This is a standard UK house with the drive in front of the house  and about 10m to the road.

First question was around placement.   In order to get the best detection is there an optimal placement of the camera (There is talk about getting better detection with movement across the camera rather than towards it).

Other question is about lighting.   The main reason for the camera initially was actually for the floodlights and I really want to keep them but not sure of the impacts on detection and and how best to use them.    The simplest option I guess is something like the Reolink Duo Floodlight which would be a like for like replacement but my experience of Reolink is that night vision isn't great (but I guess the floodlight would resolve that) but I would need to place the camera where the current one is now.   The other option is to use Home Assistant to trigger floodlights separately based on the camera detection but I figure that this needs a camera with good night detection and I wasn't sure the floodlight coming on after the detection would then skew the recording / detection.

Any advice on placement or camera choice (UK based) would be appreciated

**Comment by NickM-27** [comment]
in general placement doesn't seem like a huge issue in terms of orientation. The most difficult angle is when the camera is looking down at people from second stories and stuff. If it is closer to the top of a first story then it works well in my experience. 

I have found cameras with LEDs built in can be pretty bright, I use frigate to turn on lights via Home Assistant when a person enters a zone of interest

**Comment by hawkeye217** [comment]
There are lots of resources available on the web for camera placement techniques - do some searching for "DORI" / "detection, observation, recognition, identification)". One thing to keep in mind with camera placement that specifically has to do with Frigate is that the default model is trained on a dataset that is not weighted to the perspective of security cameras, especially when placed high above subjects looking down on them. Frigate+ is much better in this regard. 

Nonetheless, even if a person was detected, I'd still want to be able to identify their face. So I'd personally opt for placement that would be high enough to get a good overview but low enough to still be able to see faces well.
